### PR TITLE
✨ Genomic file model and tests

### PR DESCRIPTION
**Pull request by znatty22**
Add genomic file sqlalchemy model and unit tests

* Includes placeholder for sequencing experiment model and necessary updates to aliquot model. These will be overwritten once sequencing experiment model branch is merged into master and master is merged in here.

### CODE REVIEW

File: dataservice/api/genomic_file/models.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    :param uuid: UUID assigned to file from Gen3
+    :param file_name: Name of file
+    :param file_type: Type of genomic file (i.e. aligned reads)
+    :param file_format: Format of file
+    :param file_url: Location of file
+    :param md5sum: 128 bit md5 hash of file
+    """
+
+    __tablename__ = 'genomic_file'
+    uuid = db.Column(db.String(36), unique=True)
```
Comment (znatty22) [inline dataservice/api/genomic_file/models.py]: What used to be dos_id or gen3 uuid. Per @allisonheath we are using uuid to store the gen3 uuid for now.

Comment (dankolbman) [reply dataservice/api/genomic_file/models.py]: Can't this just use the uuid from the `Base`?
I think we'll probably generate the uuid inside our api here, then register that uuid with gen3, not the other way around.

Comment (znatty22) [reply dataservice/api/genomic_file/models.py]: 👍

File: dataservice/api/genomic_file/models.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    """
+
+    __tablename__ = 'genomic_file'
+    file_name = db.Column(db.Text())
+    file_type = db.Column(db.Text())
+    file_format = db.Column(db.Text())
+    file_url = db.Column(db.Text())
+    # TODO Change to use UUID for md5sum later on
+    # See link for why md5sum should use uuid type
+    # https://dba.stackexchange.com/questions/115271/what-is-the-optimal-data-type-for-an-md5-field
```
Comment (dankolbman) [inline dataservice/api/genomic_file/models.py:30]: Interesting. I never would have guessed that.

**Review by dankolbman — APPROVED** [review]
lgtm

**Review by parimalak — APPROVED** [review]
:ok_hand:

**Pull request merged**